api/v1alpha1/values_types_extra.go

```diff
@@ -93,7 +93,7 @@ type ZTunnelConfig struct {
 	// Same as `global.logging.level`, but will override it if set
 	Logging *GlobalLoggingConfig `json:"logging,omitempty"`
 	// Specifies whether istio components should output logs in json format by adding --log_as_json argument to each container.
-	LogAsJSON *bool `json:"logAsJSON,omitempty"`
+	LogAsJson *bool `json:"logAsJson,omitempty"` //nolint
```

Comment:
@luksa @dgn If I do not add `//nolint` here, I'm getting linter errors as shown below. Is this fine or do you have any suggestions?

```
INFO [runner] linters took 6.119409605s with stages: goanalysis_metalinter: 6.108544436s 
api/v1alpha1/values_types_extra.go:96:2: var-naming: struct field LogAsJson should be LogAsJSON (revive)
	LogAsJson *bool `json:"logAsJson,omitempty"`
	^
api/v1alpha1/values_types_extra.go:121:2: var-naming: struct field LogAsJson should be LogAsJSON (revive)
	LogAsJson *bool `json:"logAsJson,omitempty"`
	^
```

Reply:
How about the following:
```suggestion
	LogAsJSON *bool `json:"logAsJson,omitempty"`
```

This way, we stick to go standards and also ensure that the JSON field name matches the charts. But we should also then do the same in [value_types.go](https://github.com/istio-ecosystem/sail-operator/blob/6fb050a38ca14cf5f1be050fd06fc0f59496b6e2/api/v1alpha1/values_types.gen.go#L287) and anywhere else there's a similar issue (not just for JSON, but other acronyms as well).

We should also

Reply:
I'm not seeing any linter errors for the `values_types.gen.go` file (which is auto-generated as you know). I was only seeing the linter error for this file. Let me update it.